### PR TITLE
[Docs] add docker cluster guide

### PR DESCRIPTION
**Pull request by liunaijie**
### Purpose of this pull request

add docker cluster startup guide

### How was this patch tested?
tested in local

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by liunaijie** [comment]
image



image

File: docs/en/start-v2/docker/docker.md [hunk 1]
```diff
@@ -139,3 +137,122 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
...
+```
+![docker-network-gateway.png](../../../images/docker-network-gateway.png)
+
+3. start the nodes
+```shell
+## start master and export 5801 port 
+docker run -d --name seatunnel_master \
+    --network seatunnel-network \
+    --rm \
+    -e ST_DOCKER_MEMBER_LIST=172.18.0.1:5801 \ # set to the Gateway IP
```
Comment (Hisoka-X) [inline docs/en/start-v2/docker/docker.md]: Can we use host ip?

File: docs/en/start-v2/docker/docker.md [hunk 2]
```diff
@@ -139,3 +138,227 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
...
+The `docker-compose.yaml` file is :
+```yaml
+version: '3.8'
+
+services:
+  master:
+    image: apache/seatunnel
+    container_name: seatunnel_master
+    environment:
+      - ST_DOCKER_MEMBER_LIST=172.16.0.2,172.16.0.3,172.16.0.4
```
Comment (Hisoka-X) [inline docs/en/start-v2/docker/docker.md:229]: where can user get `ST_DOCKER_MEMBER_LIST` value?

File: docs/en/start-v2/docker/docker.md [hunk 3]
```diff
@@ -139,3 +138,227 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
...
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.16.0.0/24
+
+```
+
+and run `docker-compose up -d` command, the new worker node will start, and the current node won't restart.
+
+## 2. Use Docker Directly
```
Comment (Hisoka-X) [inline docs/en/start-v2/docker/docker.md]: How about let this as first way?

File: docs/en/start-v2/docker/docker.md [hunk 4]
```diff
@@ -139,3 +138,227 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
...
+
+run this command to start worker node.
+```shell
+docker run -d --name seatunnel_worker_1 \
+    --network seatunnel-network \
+    --rm \
+    -e ST_DOCKER_MEMBER_LIST=172.18.0.2:5801 \ # set master container ip to here
+    apache/seatunnel \
+    ./bin/seatunnel-cluster.sh -r worker
+```
```
Comment (Hisoka-X) [inline docs/en/start-v2/docker/docker.md]: Please add some shell about how to submit job? how to cancel job? how to list job?

File: docs/en/start-v2/docker/docker.md [hunk 5]
```diff
@@ -139,3 +138,227 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
...
+      seatunnel_network:
+        ipv4_address: 172.16.0.5        # use a not used ip
+
+networks:
+  seatunnel_network:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.16.0.0/24
+
```
Comment (Hisoka-X) [inline docs/en/start-v2/docker/docker.md:363]: Please add some shell about how to submit job? how to cancel job? how to list job?

File: docs/en/start-v2/docker/docker.md [hunk 6]
```diff
@@ -139,3 +138,227 @@ docker run --rm -it apache/seatunnel bash -c '<YOUR_FLINK_HOME>/bin/start-cluste
...
+docker inspect master-1
+```
+run this command to get the pod ip.
+
+- start worker node
+```shell
+docker run -d --name seatunnel_worker_1 \
+    --network seatunnel-network \
+    --rm \
+    -e ST_DOCKER_MEMBER_LIST=172.18.0.2:5801 \ # set master container ip to here
```
Comment (Hisoka-X) [inline docs/en/start-v2/docker/docker.md]: Can we directly use host ip?

File: docs/en/start-v2/docker/docker.md (hunk 2 above)
Comment (liunaijie) [reply docs/en/start-v2/docker/docker.md:229]: in docker-compose, we can define each container's ip. so we can know the ip list and set it to here

File: docs/en/start-v2/docker/docker.md (hunk 3 above)
Comment (liunaijie) [reply docs/en/start-v2/docker/docker.md]: Ok

File: docs/en/start-v2/docker/docker.md (hunk 4 above)
Comment (liunaijie) [reply docs/en/start-v2/docker/docker.md]: Ok

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @liunaijie !

**Review by hailin0 — APPROVED** [review]

**Pull request merged**